### ISSUE TITLE
Mobile password requirements and web password requirements don't match

### ISSUE THREAD

**Issue by aykutkantas** [SWE574-Fall2023-Group1/SWE574-Fall2023-G1#96]
Mobile password requires 6 character while web does not require. Thus, a web user who has defined a password with 8 character can not login to mobile app. We need to fix it to match.

**Comment by oguzsenna** [comment]
I think we can discard the mobile app password validation. WDYT?

**Comment by aykutkantas** [comment]
@oguzsenna I agree! we can discard the mobile app pass validation requirements so it can be same as web.

**Comment by sadikkuzu** [comment]
Hi @aykutkantas
What is the next move in this task?:)

**Comment by aykutkantas** [comment]
I have assigned mobile team for this issue. @mert-aydin @ayhncvdr

**Comment by sadikkuzu** [comment]
@aykutkantas hi,
I think, it's better to close this issue,
and to open a new issue on mobile repo, imho.
